rct229/rulesets/ashrae9012019/section6/section6rule5.py

```diff
@@ -185,8 +186,17 @@ def rule_check(self, context, calc_vals=None, data=None):
                         schedule_comparison_result = calc_vals[
                             "schedule_comparison_result"
                         ]
-
                         return (
                             schedule_comparison_result["total_hours_compared"]
                             == schedule_comparison_result["total_hours_matched"]
                         )
+
+                    def is_tolerance_fail(self, context, calc_vals=None, data=None):
+                        schedule_comparison_result = calc_vals[
+                            "schedule_comparison_result"
+                        ]
+
+                        return std_equal(
+                            schedule_comparison_result["total_hours_compared"],
+                            schedule_comparison_result["total_hours_matched"],
+                        )
```

Comment:
total hours compared data type is integer. We should skip the tolerance check on integer.